api/swagger.yaml

```diff
@@ -6302,6 +6315,75 @@ definitions:
     additionalProperties:
       type: "string"
 
+  ImageManifestSummary:
+    x-nullable: false
+    description: |
+      ImageManifestSummary represents a summary of an image manifest.
+    type: "object"
+    required: ["Id", "Descriptor", "Available", "ContentSize", "Kind"]
+    properties:
+      Id:
+        description: |
+          ID is the content-addressable ID of an image and is the same as the
+          digest of the image manifest.
+        type: "string"
+        example: "sha256:95869fbcf224d947ace8d61d0e931d49e31bb7fc67fffbbe9c3198c33aa8e93f"
+      Descriptor:
+        $ref: "#/definitions/OCIDescriptor"
+      Available:
+        description: Indicates whether all the children content is available.
+        type: "boolean"
+        example: true
```

Comment:
Is there a use case for consumers to know _which_ child content is available? (ie, we have the config object, the specific layer that contains the provenance doc we're interested in, or something similar, but we don't have _all_ of them?)

Reply:
Not sure, I think it's a bit out of scope of the "image list". Perhaps exposing such information would make more sense in "image inspect"?